File: perl/fizzbuzz.pl
use warnings;
use strict;

for ( my $i = 1; $i < 101; $i = $i + 1 ){
	if($i % 3 eq 0 && $i % 5 eq 0){
		print("fizzbuzz");
		print("\n");
	}elsif($i % 3 eq 0){
		print("fizz");
		print("\n");
	}elsif($i % 5 eq 0){
		print("buzz");
		print("\n");
	}else{
		print($i);
		print("\n");
	}
}

File: fsharp/fizzbuzz.fs
let seq1 = seq { for i in 1 .. 100 -> (i)}
for (x) in seq1 do
  if x % 3 = 0 && x % 5 = 0 then printfn "fizzbuzz"
  elif x % 3 = 0 then printfn "fizz"
  elif x % 5 = 0 then printfn "buzz"
  else printfn "%i" x